### ISSUE TITLE
Upgrade our ormolu

### ISSUE THREAD

**Issue by srid** [nammayatri/nammayatri#737]
We're using an old version of ormolu (0.1.4.1). Either upgrade to the latest, or switch to fourmolu.

**Comment by aravindgopall** [comment]
+1 for fourmolu and `,` at end.

**Comment by srid** [comment]
Yes, fourmolu is my preference as well. We can use this config: https://github.com/srid/haskell-template/blob/master/fourmolu.yaml

@hkmangla @NikithShetty What do you think? It'd improve the DX a bit, since you get auto-format from HLS.

**Comment by arjunkathuria** [comment]
i just ran into something and ended up updating ormolu to the newest version as a part of upgrading the entire nammayatri project to GHC 9.2.7 ecosystem. 

PR coming soon.

**Comment by srid** [comment]
Fourmolu config is now nixified, and available as flake-parts module: https://github.com/jedimahdi/fourmolu-nix

Example usage: https://github.com/srid/haskell-template/commit/49a020bf71153b5fdc9fb46e3521f94501718b4c
